bot/__main__.py

```diff
@@ -51,6 +51,7 @@
 bot.load_extension("bot.cogs.information")
 bot.load_extension("bot.cogs.jams")
 bot.load_extension("bot.cogs.moderation")
+bot.load_extension("bot.cogs.news")
```

Comment:
This needs changing to the new name
```suggestion
bot.load_extension("bot.cogs.python_news")
```